1735Sprint/04 - Xander/Demo.java

```diff
@@ -0,0 +1,28 @@
+import character.*;
+import java.util.*;
+
+public class Demo {
+  final static String BREAK = "\n";
+  List<CharacterBase> charList = new ArrayList<CharacterBase>();
+
+  public static void main(String[] args) {
+    Demo dm = new Demo();
+    dm.executeAllMethods();
+
+  }
+
+  public Demo() {
+    Crusader velstadt = new Crusader("Velstadt");
+    Necromancer elana = new Necromancer("Elana");
+    DragonKnight jan = new DragonKnight("Jan");
```

Comment:
You can remove line 15 ~ 17 since it's unused. You already create these objects in line 19